SinaiStCatherine/SinaiEt004.xml

```diff
@@ -471,32 +472,34 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                 <list>
                                   <item xml:id="a1">
                                     <locus target="#1v"/>
-                                    <desc type="Unclear">Unclear note, seems to indicated the dimensions of the body of Adam.</desc>
+                                    <desc type="Unclear">An note indicating the great stature of <persName ref="PRS1386Adam">Adam</persName></desc>
                                     <q xml:lang="gez">
-                                      </q>
-                                    <!-- try to read in ms.-->
-                                  </item>
+                                    </q><supplied reason="undefined">፲ወ፯፼፡ ፷፻፡ ፯፻፡ ፯፻፡ ፼፷፻፡ ፲ወ፬፻፡ ፳፡ </supplied>ወእነግር፡ ትካ<supplied reason="lost">ተ፡</supplied>
+                                    በነገረ፡ ጽሑፍ፡ ቆሙ፡ ለአዳም፡ ፼፡ ፺፻፡ ወ፮፻። ስ<supplied reason="omitted" resp="DN">ፍ</supplied>ሐ፡ 
+                                    እንግድዓሁ፡ ፴፻፡ ወ፫፡ ፷፡ መሬት፡ ዘዱዳሌም፡ ወማይ፡ ዘኤዶም፡ ቆሙ።
+                                     </item>
                                   
                                   
                                 <item xml:id="a2">
-                                  <locus from="2r" to="3r"/>
-                                  <desc type="GuestText"><title ref="LIT2457Testam" type="incomplete"></title> 
-                                    A few initial lines 
-                                    are missing. Three last lines of the 
-                                  text are written in the bottom margin of the folium. </desc>
-                                  <q xml:lang="gez"><gap reason="illegible"/>ኦወልድየ፡ በቀዳሚት፡ ሰዓተ፡ 
+                                  <locus from="2r" to="4r"/>
+                                  <desc type="GuestText"><title ref="LIT1957Mashaf" type="incomplete">An excerpt from the <foreign xml:lang="gez">Maṣḥafa Qalemǝnṭos</foreign></title></desc>
```

Comment:
It is better to move "An excerpt from the" in front of the title element, and to have no markup inside the title element